docker/linux/base/build/Dockerfile

```diff
@@ -22,12 +22,19 @@ RUN apt update -qqy \
     && apt install -qqy --no-install-recommends \
         binutils \
         bison \
-        nasm \
         ninja-build \
         pkgconf \
+        software-properties-common \
         unzip \
     && /opt/clean-after-apt.sh
 
+# === Install nasm 2.16 using backport from PPA
```

Comment:
The code seems correct but this should only go to old Ubuntu images, not unconditionally in base image

Reply:
E.g the file you want this in is probably https://github.com/youtube/cobalt/blob/main/docker/linux/gcc-6-3/Dockerfile

Reply:
Ack, moved the files